pypeit/scripts/chk_edges.py

```diff
@@ -22,7 +22,13 @@ def parser(options=None):
 
     return parser.parse_args() if options is None else parser.parse_args(options)
 
-
+# TODO: JFH I don't see why we are showing the edges and/or what the
+# purpose of all this if synced not synced is fore. edgetrace seems to
+# crash if the syncing fails. So if we have successfuly run EdgeTrace,
+# we create a slittrace object and the slittrace object is the thing we
+# should be showing not the edgetrace object. This has the advantage
+# that then orders are correctly labeled for Echelle which is not the
+# case with the current show method.
```

Comment:
@kbwestfall I'm not sure where you would ever run the show method if you were not synced. That would presumably be during debugging in which case you could just embed into the code and show the slits as you like. The edgetrace method crashes if syncing fails, as far as my experience. You are claiming it can happen, I'm saying it never does. And so as far as I can tell all the code checking for syncing vs not syncing in the edgetrace show method has no value, since it is never used, but it makes the code a lot harder to understand and read. I'm advocating you assume syncing to show the slits, because the logic in the current show method is in my opinion unnecessarily complicated.

Reply:
Change it as you like.  If you do, please check that `pypeit_trace_edges` runs with the `--show` option or remove the option.